lib/slices/tree.rb

```diff
@@ -39,7 +39,7 @@ module Tree
       field :position, type: Integer
       field :show_in_nav, type: Boolean, default: false
 
-      scope :minimal, ->{ only(%w(_type page_id path external_url name has_content)) }
+      scope :minimal, ->{ only(%w(_id _type page_id path external_url name has_content)) }
```

Comment:
Commits says

> We need ids to generate links

Can you explain a bit more? Interested to know how it worked before now.